### ISSUE TITLE
Speed up the calculation of norms

### ISSUE THREAD

**Issue by UnaiSan** [SINTEF/Splipy#142]
As it is shown [here](https://stackoverflow.com/a/45006970), there is an improvement in the calculation of norms if instead of

```{.python}
np.apply_along_axis(np.linalg.norm, -1, array)
```

we write

```{.python}
np.linalg.norm(array, axis=-1)
```

I have seen this in the following functions/methods: `Curve.binormal`, `Curve.curvature`, `Curve.torsion`, `SplineObject.tangent`, `Surface.normal`.

Could this be addressed in a future release?

**Comment by VikingScientist** [comment]
This is a great suggestion. I was not aware of this discrepancy.  Thank you for bringing this to our attention.

**Comment by VikingScientist** [comment]
Fixed #144
